features/buyers/accounts/new.feature

```diff
@@ -37,3 +37,22 @@ Feature: Audience > Accounts > New
     And field "Email" has inline error "should look like an email address"
     And field "Organization/Group Name" has inline error "can't be blank"
     But field "Password" has no inline error
+
+  @javascript
+  Scenario: Create account with fields with choices
+    Given the provider has the following fields defined for accounts:
+      | name                 | choices                 | label                  | required | read_only | hidden |
+      | country              |                         | Country                | true     |           |        |
+      | field_with_choices   | hello, option1, option2 | Fields with choices    | true     |           |        |
+    And they go to the new buyer account page
+
+    When they select "Spain" from "Country"
+    And they select "option1" from "Fields with choices"
+    And the form is submitted with:
+      | Username                | alice              |
+      | Email                   | alice@example.com  |
+      | Organization/Group Name | Alice's Web Widget |
+
+    Then the current page is the buyer account page for "Alice's Web Widget"
+    And they should see "Spain"
+    And they should see "option1"
```

Comment:
This approach is "the old way" because the assertion is to broad. I suggest asserting a table within a selector OR adding this to features/old/accounts/buyer_account.feature
```suggestion
    And they should see the following table within the account details card:
      | Country | Spain |
      | Fields with choices | option1 |
```

Reply:
Addressed in https://github.com/3scale/porta/pull/3685/commits/4188fd6a2c385e7854dbc3e7d9ece8f7e8d6c2d2